symbols/overlay29.yml

```diff
@@ -2208,6 +2208,22 @@ overlay29:
         
         r0: Pointer to an entity. Can be null.
         returns: Seems to be true if there were any pending actions to display.
+    - name: EndSleepClassStatus
```

Comment:
If we know the parameters for these functions, don't forget to update the function signatures in `headers/functions/overlay29.h`